src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -65,7 +101,15 @@ const CurriculumDownloadTab: FC<CurriculumDownloadTabProps> = ({
         schools: [],
       });
     }
-  }, [localStorageData]);
+
+    // Set the subject tier selector as visible when tiers & child_subjects are present
+    if (
```

Comment:
Any reason this needs to be in the same `useLayoutEffect` hook?

Reply:
I had them initially in two different useLayoutEffects - but thought it looked a bit neater. Would it make more sense for there to be two separate ones?